app/queries/tweets_for_month.rb

```diff
@@ -0,0 +1,21 @@
+class TweetsForMonth < Patterns::Query
+  queries Tweet
+
+  private
+
+  def query
+    relation.where(tweet_created_at: date_range, organization: organization)
+  end
+
+  def organization
+    options.fetch(:organization)
+  end
+
+  def date
+    options.fetch(:date, Date.today).to_datetime
+  end
+
+  def date_range
+    date.beginning_of_month..date.end_of_month
```

Comment:
check if that will fetch a tweet from the last day of the month

Reply:
I think we don't need this file anymore